src/wallet-account.ts

```diff
@@ -230,6 +230,10 @@ class ConnectedWalletAccount extends Account {
 
         if (permission.FunctionCall) {
             const { receiver_id: allowedReceiverId, method_names: allowedMethods } = permission.FunctionCall;
+            // check if access_key has 2fa method
```

Comment:
I think it's worth adding more expanded comment / linking to multisig contract code.